### ISSUE TITLE
Delete excluded rules for `duplicate-id` and `duplicate-id-active` from the component guide accessibility tests

### ISSUE THREAD

**Issue by jon-kirwan** [alphagov/govuk_publishing_components#4276]
## What

Delete the excluded rules for `duplicate-id` and `duplicate-id-active` from the component guide accessibility tests.

## Why

> The [4.8 release](https://github.com/dequelabs/axe-core/releases/tag/v4.8.0) of Axe Core deprecated the `duplicate-id` and `duplicate-id-active` rule, along with setting the `duplicate-id-aria` to always report as "needs review" rather than as a violation. 

> ...Axe-core currently has three rules to test success criterion 4.1.1 Parsing. All three of these are related to duplicate IDs

For WCAG 2.2 criterion 4.1.1 Parsing is removed.

https://www.deque.com/blog/wcag-2-2-removes-4-1-1-parsing-and-how-axe-core-is-impacted/.
